src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/CollectionsMarshalTests.cs

```diff
@@ -299,6 +299,200 @@ public void GetValueRefOrNullRefLinkBreaksOnResize()
             Assert.Equal(50, dict.Count);
         }
 
+        [Fact]
+        public void GetValueRefOrAddDefaultValueType()
+        {
+            // This test is the same as the one for GetValueRefOrNullRef, but it uses
+            // GetValueRefOrAddDefault instead, and also checks for incorrect additions.
+            // The two APIs should behave the same when values are already existing.
```

Comment:
```suggestion
            // The two APIs should behave the same when values already exist.
```

Reply:
Fixed as part of 5209a5bedff52a33454da401854502d6b967501a 👍